examples/kni/README.md

```diff
@@ -6,15 +6,21 @@ The Kernel NIC Interface (KNI) is a DPDK control plane solution that allows user
 
 KNI is useful for applications that want to conceptually share the port with the Linux kernel. For example, the application may want to leverage the kernel's built-in ability to handle [ARP](https://tools.ietf.org/html/rfc826) traffic instead of implementing the protocol natively. By enabling KNI for a port, a virtual device with the same name and MAC address as the port is exposed to the Linux kernel. The kernel will be able to receive all packets that are forwarded to this virtual device and the application will receive all packets the kernel sends to it.
 
-## Running the application
+## Prerequisite
```

Comment:
kni is the only readme that needed update. I purposely didn't link to sandbox because I want these examples to be more self contained in capsule.